### DISCUSSION TITLE
Lesson 4 can't find Kovan Faucet anymore

### DISCUSSION THREAD

**Discussion by Wildkraken** [Cyfrin/foundry-full-course-cu#184] [Q&A] [answered]
Hello, first a big thank for these courses
I'm in lesson 4 I cant find the Kovan Faucet on ChainLink
the link dosent work anymore, 
is there any alternative,
thank you

**Comment by rohit-bhalerao-7** [comment]
Why do need Kovan tokens? I believe they're deprecated. Use Sepolia instead. I hope it helps

**Reply by ChefAharoni** [reply, marked as answer]
Yep it was indeed deprecated. You should use Sepolia instead.
You can find more information in the first section of this repo's [README](https://github.com/Cyfrin/foundry-full-course-f23#testnet-faucets)
